examples/cassandra-3.11/docker-compose.yml

```diff
@@ -2,8 +2,8 @@ version: '2'
 
 services:
   backend-1:
-    image: cassandra:3.11.8
-    container_name: backend-1
+    image: cassandra:${CASSTAG}
+    #container_name: backend-1
```

Comment:
Should be commented-out line be removed perhaps?

Reply:
Why do we need to comment it out, by the way? :)

Reply:
I'll take them out. I first commented them out when I realized running multiple docker-compose files at the same time interfered with one another when the containers had the same names.